rakudoc_draft_3.rakudoc

```diff
@@ -112,6 +112,13 @@ I<Titlecase> spelling, which is considered a possible but non-canonical variant.
 
 A file with mainly RakuDoc source has a file extension B<.rakudoc>.
 
+=nested As it will become apparent, whitespace and indentation plays an important role in RakuDoc.
+There are two whitespace characters, space and tab. No equivalence is made between
```

Comment:
I think there's actually more.  Also, we're talking about "horizontal whitespace characters".  Perhaps something like:

```
There are L<quite a few horizontal whitespace characters|https://unicode-explorer.com/articles/space-characters>.
No equivalence is made between them, they are treated as separate entities.
For instance, mixing spaces  and tabs....
```